### PR TITLE
Remove creation of FixedSource instance from class Feedin

### PR DESCRIPTION
**Pull request by birgits**

### CODE REVIEW

File: oemof/db/feedin_pg.py [hunk 1]
```diff
@@ -19,8 +18,28 @@ def __init__(self):
...
+        ----------
+        region : Region instance
+        region.geom : shapely.geometry object
+            Geo-spatial data with information for location/region-shape. The
+            geometry can be a polygon/multi-polygon or a point.
+
+        Returns
+        -------
+        feedin_df : pandas dataframe
+            Dataframe containing the normalised feedin profile of the given
```
Comment (uvchik) [inline oemof/db/feedin_pg.py:36]: I do not see where you normalise the feedin profile. I thought there would be something like 

``` python
for source_type in feedin_df.keys():
    feedin_df[source_type] = feedin_df[source_type] / cap[source_type]
```

Or do I misunderstand something?

**Comment by birgits** [comment]
Sorry, I must have accidentally deleted it when I reverted a commit... Should be alright now!

**Comment by uvchik** [comment]
Looks fine now but I didn't check the results.

Did you check the results? Are they plausible? If both answers are "yes" feel free to merge.

**Comment by birgits** [comment]
Yes and yes :)

**Pull request merged**